spec/models/timeslot_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Timeslot, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :start_time }
+    it { is_expected.to have_db_column :end_time }
+  end
+
+  describe 'Associations' do
+    it { is_expected.to belong_to :facility }
+  end
+
+  describe 'Factory' do
+    it 'should have valid Factory' do
+      expect(create(:timeslot)).to be_valid
+    end
+  end
```

Comment:
Looking at this method it doesn't look like something I'd want to have in the model, to begin with 🤔 
The fact that we're creating new instances of `Timeslot` within that method suggests that it should be outside of this. A controller perhaps?